src/planwise/client/components/nav.cljs

```diff
@@ -3,12 +3,17 @@
 
 ;; Navigation components
 
-(defn- li-menu-item [{:keys [selected item href title icon]}]
+(defn- li-menu-item [{:keys [selected item href title icon tab-number wizard-state]}]
   (let [is-selected? (or (= selected item)
                          (item selected))]
-    [:li {:class (when is-selected? "active")}
+    [:li {:class (str
+                  (when is-selected? "active ")
+                  (when wizard-state (str (name wizard-state) " wizard")))}
      [:a {:href href}
       (some-> icon (components/icon "icon-small"))
+      (when (and wizard-state (> tab-number 1)) [:div.separator])
```

Comment:
The word redundant is probably not the most adequate. Since it's a strictly visual element, it can be done entirely with CSS. Check https://jsfiddle.net/pxc0n8vj/ :)

If that's not enough, you'd be better off using `interpose` on the parent. See http://clojuredocs.org/clojure.core/interpose